### ISSUE TITLE
Docs erroneously claim that the Recover JobWrapper is added by default

### ISSUE THREAD

**Issue by alext** [robfig/cron#380]
Hi,

As far as I can tell from the code, the Recover JobWrapper is not added by default despite the documentation saying it is.  Commit d279950 would suggest that not adding it by default is the intended behaviour.

Can the docs be updated to match the implementation (I spotted this in both the overview Job Wrappers section, and `cron.New` function docs)?

thanks

**Comment by robfig** [comment]
Agreed, it was removed as a default without updating the documentation. Thanks for catching that.
